### DISCUSSION TITLE
smtp problem

### DISCUSSION THREAD

**Discussion by gajosew** [F43nd1r/Acrarium#322] [General]
I am trying to use the sending of e-mails. and unfortunately :

acrarium    | org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mailService': Injection of autowired dependencies failed; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'spring.mail.sender' in value "${spring.mail.sender}"

**Comment by F43nd1r** [comment]
See https://github.com/F43nd1r/Acrarium/issues/300, fix released in 1.7.0

**Reply by gajosew** [reply]
in wiki :  spring.mail.host=localhost

**Reply by gajosew** [reply]
it ok. THX
